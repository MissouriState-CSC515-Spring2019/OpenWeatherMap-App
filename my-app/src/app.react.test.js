import React from 'react';
import componentConverTemp from '../src/components/CurrentWeather/Weather';
import renderer from 'react-test-renderer';

test('Check to make sure the DOM tree is what is rendered', () => {
  const component = renderer.create(
    <componentConverTemp />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});