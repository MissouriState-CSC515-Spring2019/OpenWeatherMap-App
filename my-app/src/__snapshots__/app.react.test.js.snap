// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check to make sure the DOM tree is what is expected. 1`] = `<componentConverTemp />`;

exports[`Check to make sure the DOM tree is what is rendered 1`] = `<componentConverTemp />`;

exports[`Link changes the class when hovered 1`] = `<componentConverTemp />`;
